playbooks/openstack-caasp_instance.yml

```diff
@@ -180,6 +180,7 @@
             path: "{{ socok8s_extravars }}"
             regexp: "^socok8s_ext_vip.*"
             line: "socok8s_ext_vip: {{ _vipip.stdout }}"
+            create: yes
```

Comment:
While I agree this would be fixing the parallel work, running this in parallel could technically enter a race condition where this task run at the same time as the templating of ses osh vars, which could then cause of or the other to get non-written. Same applies to inventory patching above.

These plays are not meant for running in parallel. I don't think it's a wise idea to do this in parallel, I am not sure how much we gain by doing so....

I am not strongly against, but for other reviewers, keep in mind adding this parameter basically opens us to race conditions.

Reply:
If we make those changes on different files, I am in.

Reply:
I think we should have an assert at the end of the play that checks the content of the extra vars, to ensure the vip was well set (by doing a lookup('file',...)) . That could work enough for me, and ensure the file has the vip.